### ISSUE TITLE
Export/Import react-payment-inputs/images (documentation error?)

### ISSUE THREAD

**Issue by agrohs** [medipass/react-payment-inputs#6]
Not sure if we're doing something wrong here and/or need to do an additional yarn install, but when trying to use the PaymentInputsWrapper, we are having an issue w/ basic import of the credit card images: `import images from 'react-payment-inputs/images';`.

It gives us an error of:
```
Cannot find module: 'react-payment-inputs/images'. Make sure this package is installed.

You can install this package by running: yarn add react-payment-inputs/images.
```

Looked inside of the installed node_modules directory and see the `react-payment-inputs/es (and /lib and /umd)` folders - with the /es and /lib ones each containing the subfolder for images but not sure how they are exported in there via rollup such that the images can be imported properly??

Any idea on what we're doing wrong?

Do we need to explicitly import the es6 version? (`import images from 'react-payment-inputs/es/images';`)

**Comment by efstathiosntonas** [comment]
```
import  images from 'react-payment-inputs/lib/images';
```

then
```
 <PaymentInputsContainer>
                        {({
                          meta,
                          getCardNumberProps,
                          getExpiryDateProps,
                          getCVCProps,
                            getCardImageProps
                        }) => (
                          <div>
                            <svg {...getCardImageProps({ images })} />

```

**Comment by dkbuntzen** [comment]
I also had this issue @agrohs

I had some luck with:
```
import images from "react-payment-inputs/lib/images"
```

**Comment by jxom** [comment]
Should be fixed in `1.0.6`
